lute/book/forms.py

```diff
@@ -18,7 +18,7 @@ class NewBookForm(FlaskForm):
 
     language_id = SelectField("Language", coerce=int)
 
-    title = StringField("Title", validators=[DataRequired(), Length(max=255)])
+    title = StringField("Title", validators=[Optional(), Length(max=255)])
```

Comment:
(This is the section I was referring to when I said that this change felt incorrect)

Reply:
Sure, this was my mistake. I first made the changes through the backend and then switched to frontend so I missed this issue. I will resolve this asap.

Reply:
Cheers thanks @Jayanth-Parthsarathy :-) yeah I think this is all front-end-only.  Good idea with the js handler, much better than my bad idea!